output.tf

```diff
@@ -25,3 +25,7 @@ output "alb_security_group_id" {
 output "cicd_user_name" {
   value = "${aws_iam_user.cicd.name}"
 }
+
+output "ecs_app_role_id" {
+  value = ${aws_iam_role.app_role.id}
```

Comment:
Well, you can just use `aws_iam_role.app_role.id`, I don't think that `${aws_iam_role.app_role.id}` is a valid syntax without interpolation.

Reply:
This is probably correct, as the build that's using this module is failing because of this, I'll open a new PR asap.

Reply:
I really dislike the fact that this repo doesn't have any CI checks and doesn't make it easy to catch errors like this.